src/pages/library/common.js

```diff
@@ -166,7 +167,9 @@ export const Sidebar = ({ onSectionFilter, onTagFilter, sections, selectedSectio
   ])
 }
 
-export const selectionActionComponent = (selectedData, setSelectedData) => {
+export const SelectionActionComponent = (selectedData, setSelectedData) => {
```

Comment:
👍🏼  on capitalizing this correctly. We do also need to pass the inputs as part of an object in order to call it with `h`, like:

```suggestion
const SelectionActionComponent = ({ selectedData, setSelectedData }) => {
```

Removed `export` since this component is only used within this file.